README.md

```diff
@@ -43,11 +49,28 @@ change as part of the work.
     UserInputAdvanced("User Input Advanced 🦸‍♀️")
 ```
 
-## Required Environment Variables
+## Developer notes
+
+### Confirming functionality after dependency updates
+
+This application has test coverage >95%, so running the test suite is likely sufficient to confirm
+functionality in most cases. Some click testing is also useful, particularly if there have been
+updates to `graphql-ruby` or `graphql-client`:
+
+1. Confirm that basic and advanced searches do not error and return results.
+2. Confirm that geospatial searches (bounding box and distance) do not error and return results.
```

Comment:
Do you think it would be helpful to add a note that this requires testing against an index that has geospatial data loaded? I'm waffly as I feel like that is obvious, but I'm also wondering if in the future we may be confused without "obvious" reminders like that once the freshness of the geo data has worn off.

Reply:
Yeah, that's a good point.